src/report_batch.h

```diff
@@ -16,11 +16,12 @@
 #ifndef MIXERCLIENT_REPORT_BATCH_H
 #define MIXERCLIENT_REPORT_BATCH_H
 
-#include <mutex>
-
 #include "include/client.h"
 #include "src/attribute_compressor.h"
 
+#include <atomic>
+#include <mutex>
```

Comment:
do we need mutex?

Reply:
Yes, we have std::mutex mutex_ at line 63.